packages/openai-compatible/README.md

```diff
@@ -81,4 +81,4 @@ const { text } = await generateText({
 });
 ```
 
-For more examples, see the [OpenAI Compatible Provider](/providers/ai-sdk-providers/openai-compatible) documentation.
+For more examples, see the [OpenAI Compatible Providers](https://sdk.vercel.ai/providers/openai-compatible-providers) documentation.
```

Comment:
prefer relative links so it works locally

Reply:
This is in the npm README though, I think we need a fully qualified url? I missed it too the first time.